Source/Core/Core/WiiUtils.cpp

```diff
@@ -461,10 +461,11 @@ OnlineSystemUpdater::Response OnlineSystemUpdater::GetSystemTitles()
   std::string base_url = Config::Get(Config::MAIN_WII_NUS_SHOP_URL);
   if (base_url.empty())
   {
-    // Note: We don't use HTTPS because that would require the user to have
-    // a device certificate which cannot be redistributed with Dolphin.
-    // This is fine, because IOS has signature checks.
-    base_url = "http://nus.shop.wii.com";
+    // The NUS servers for the Wii are currently offline as of PR 10634, but the backing CDN is
+    // still active and accessible from other URLs. We take advantage of this by hosting our own
+    // NetUpdateSOAP endpoint which serves the correct list of titles to install along with URLs
+    // for the Wii U CDN.
+    base_url = "https://fakenus.dolphin-emu.org";
```

Comment:
Because of https://bugs.dolphin-emu.org/issues/11772, you must ifdef this to use HTTP on Android.